### PR TITLE
Solution [Danylo Kunchenko]

### PR DESCRIPTION
**Pull request by d-kunchenko**

### CODE REVIEW

**Review by bogdank99 — APPROVED** [review]
good job

File: cypress/e2e/bookStore.cy.js [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
+  });
+
+  it('should add book', () => {
+    cy.get('#gotoStore').click();
+    cy.get('#searchBox').should('exist').type('Speaking JavaScript');
+    cy.contains('a', 'Speaking JavaScript').click();
+    cy.get('#description-label').should('exist');
+    cy.get('.text-right > #addNewRecordButton').click();
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal(`Book added to your collection.`)
```
Comment (tnboyko) [inline cypress/e2e/bookStore.cy.js:24]: Go to your profile page and assert 'Speaking JavaScript' in your shopping list.

File: cypress/e2e/bookStore.cy.js [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
   });
+});
 
-  it('', () => {
-    
+  it('should delete book', () => {
+    cy.url('equal','/profile');
+    cy.contains('a', 'Speaking JavaScript').should('exist');
```
Comment (tnboyko) [inline cypress/e2e/bookStore.cy.js:30]: right now your test depends on the previous one, add another book here and then delete it

File: cypress/support/commands.js [hunk 3]
```diff
@@ -23,3 +23,13 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('login', () => {
+  const username = 'Snow228';
+  const password = 'Snow228!';
+
+  cy.get('#userName').should('exist').type(username);
```
Comment (tnboyko) [inline cypress/support/commands.js:31]:
```suggestion
  cy.get('#userName').type(username);
```
no need to check the existence, if there is no such field cypress won't be able to type in it

**Review by tnboyko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**